source/administration/changelog.md

```diff
@@ -2,11 +2,18 @@
 
 This changelog summarizes updates to [Mattermost Team Edition](http://www.mattermost.org/), an open source team messaging solution released bi-monthly under an MIT license, and [Mattermost Enterprise Edition](https://about.mattermost.com/pricing/), a commercial upgrade offering enterprise messaging for large organizations.
 
-## Release v3.7.0
+## Release v3.7.1
 
-- Release date: 2017-03-16
+### Notes on Patch Release
+
+ - **v3.7.1, released 2017-03-16**
+   - Fixed an issue where some [System Console > Policy settings](https://docs.mattermost.com/administration/config-settings.html#policy) were incorrectly applied to Team Edition, breaking the System Console UI
```

Comment:
@esethna It technically did --> if an admin sets the permissions from config.json, the Policy settings appear in the System Console (in TE) as if they're editable.

However, if you try to edit them, nothing happens as modifying the settings on the client-side doesn't work.